spring-cloud-kubernetes-core/src/main/java/org/springframework/cloud/kubernetes/profile/KubernetesProfileApplicationListener.java

```diff
@@ -47,13 +45,6 @@ public void onApplicationEvent(ApplicationEnvironmentPreparedEvent event) {
     }
 
     void addKubernetesProfile(ConfigurableEnvironment environment) {
-        Pod current = utils.currentPod().get();
```

Comment:
Is this the same as `utils.isInsideKubernetes()`?

Reply:
Yes it is (https://github.com/spring-cloud-incubator/spring-cloud-kubernetes/blob/master/spring-cloud-kubernetes-core/src/main/java/org/springframework/cloud/kubernetes/StandardPodUtils.java#L55):
```java
@Override
public Boolean isInsideKubernetes() {
    return currentPod().get() != null;
}
```